src/diffusers/models/attention.py

```diff
@@ -148,6 +162,8 @@ def __init__(
         attention_type: str = "default",
         positional_embeddings: Optional[str] = None,
         num_positional_embeddings: Optional[int] = None,
+        temporal_position_encoding: Optional[bool] = False,
```

Comment:
`positional_embeddings` and `num_positional_embedding` will handle this.

Reply:
Oh, got it. Will change.